lib/brainstem/query_strategies/README.md

```diff
@@ -0,0 +1,94 @@
+# Filtering/Searching Query Strategies
+
+Traditionally Brainstem has ignored all filters if you defined a `search` block in your presenter.
+Brainstem relies on your search implementation to do any necessary filtering. A downside of this is that you may have to
+implement your filters twice: once inside your presenters and once inside
+your searching solution. This causes extra work, particularly for complex queries and associations that the Brainstem DSL
+is well equipped to handle.
+
+The current version of Brainstem offers a beta feature for allowing both searching and filtering to take place. To enable it,
+add the following to your presenter:
+
+```ruby
+query_strategy :filter_and_search
+```
+
+Utilizing this strategy will enable Brainstem to take the intersection of your search results and your filter results,
+effectively giving you the best of both worlds: fast, efficient searching using something like ElasticSearch and in depth
+ActiveRecord filtering provided by Brainstem.
+
+You must take the following important notes into account when using the `filter_and_search` query strategy:
+
+- Your search block should behave the same as it always has: it should return an array where the first element is an array
+  of model ids and the second element is the total number of matched records.
+- This works by retrieving all possible ids from your search block (up to 10,000) and then applying your filters
+  on those returned ids. This has some obvious potential performance considerations as you are potentially returning
+  and querying off of 10,000 results.
+- If you have less than 10,000 possible results you shouldn't have to worry about ordering, because the order will
```

Comment:
I recall running some tests like `where(id: [1...20,000])` ids and seeing the MySQL was slow.  You should check and see what that performance looks like for you.